sources/packages/backend/libs/src-sql/disbursement-overawards/Bulk-insert-bcsl-disbursement-overaward.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Insert BCSL overawards from SFAS integration to disbursement overawards table where 
+ * the overard is different than 0. 
+ */
+INSERT INTO sims.disbursement_overawards(
+		student_id,
+		disbursement_value_code,
+		overaward_value,
+		origin_type,
+		creator) 
+SELECT	sfas_individuals.student_id, 
+	   	E'BCSL', 

```

Comment:
Is there a possibility to provide the query `BCSL` as a parameter and reuse the same query for both awards?
If so, maybe `system-user` and `Legacy overaward` can also be provided as parameters.

Reply:
It's possible if we create the SQL file as a template:
`AND disbursement_overawards.disbursement_value_code = ${disbursementValueCode};`
Is that what you mean?

Reply:
Not really, if I am not wrong we can use real DB parameters.
```ts
repository.query(`select * from users where LOWER(first_name) = $1 OR LOWER(last_name) = $2`, [ first_name.toLocaleLowerCase(), last_name.toLocaleLowerCase() ])
```

Reply:
ok. Added parameters. The SQL Formatter doesn't work well with parameters but it's just turn it off when changing this file.